api/features/tasks.py

```diff
@@ -82,3 +192,82 @@ def _get_feature_state_webhook_data(feature_state, previous=False):
         identity_identifier=getattr(feature_state.identity, "identifier", None),
         feature_segment=feature_state.feature_segment,
     )
+
+
+def _save_feature_state_value_with_type(
+    value: Optional[Union[int, bool, str]], feature_state_value: FeatureStateValue
+) -> None:
+    if isinstance(value, int):
+        feature_state_value.type = INTEGER
+        feature_state_value.integer_value = value
+    elif isinstance(value, bool):
+        feature_state_value.type = BOOLEAN
+        feature_state_value.boolean_value = value
+    else:
+        feature_state_value.type = STRING
+        feature_state_value.string_value = value
+
+    feature_state_value.save()
+
+
+def _create_multivariate_feature_option(
+    value: Optional[Union[int, bool, str]],
+    feature: Feature,
+    default_percentage_allocation: Union[int, float],
+) -> MultivariateFeatureOption:
+    if isinstance(value, int):
+        mvfo = MultivariateFeatureOption.objects.create(
+            integer_value=value,
+            type=INTEGER,
+            feature=feature,
+            default_percentage_allocation=default_percentage_allocation,
+        )
+    elif isinstance(value, bool):
+        mvfo = MultivariateFeatureOption.objects.create(
+            boolean_value=value,
+            type=BOOLEAN,
+            feature=feature,
+            default_percentage_allocation=default_percentage_allocation,
+        )
+    else:
+        mvfo = MultivariateFeatureOption.objects.create(
+            string_value=value,
+            type=STRING,
+            feature=feature,
+            default_percentage_allocation=default_percentage_allocation,
+        )
+    return mvfo
+
+
+def _create_new_feature(
+    rec: dict[str, Optional[Union[bool, str, int]]],
+    project: Project,
+    environment: Environment,
+) -> None:
+    feature = Feature.objects.create(
+        name=rec["name"],
+        project=project,
+        initial_value=rec["initial_value"],
+        is_server_key_only=rec["is_server_key_only"],
+        default_enabled=rec["default_enabled"],
+    )
+    feature_state = feature.feature_states.filter(
+        environment=environment,
+    ).first()
```

Comment:
It's a little odd to me that we're using `.first()` here - I think we could probably just use `.get()`, right?

Reply:
Yeah that makes sense.